src/api/sso/package.json

```
@@ -5,7 +5,10 @@
   "description": "An SSO Authentication and Authorization Service",
   "scripts": {
     "dev": "env-cmd -f env.local nodemon src/server.js",
-    "start": "node src/server.js"
+    "start": "node src/server.js",
+    "eslint": "TIMING=1 eslint --config .eslintrc.js \"./src/**/*.js\" \"./test/**/*.js\"",
+    "eslint-fix": "eslint --config .eslintrc.js \"./src/**/*.js\" \"./test/**/*.js\" --fix",
```

what about the current directory? I'm not sure about this one too, I need @DukeManh

---

Let's include every `js` file.
```suggestion
    "eslint-fix": "eslint --config .eslintrc.js \"./**/*.js\" --fix",
```